af/fastpath/fastpath/utils.py

```diff
@@ -430,9 +430,9 @@
     ],
     "PL": [
         # https://github.com/ooni/pipeline/issues/224
-        # https://explorer.ooni.org/measurement/20190911T110527Z_AS6830_kkm1ZGUCJI4dSrRV4xl6QwG77o7EeI2PDKbwt9SPL9BBJHUsTr?input=http://www.eurogrand.com/
+        # https://explorer.ooni.org/measurement/20210409T092428Z_webconnectivity_PL_12912_n1_DoMtAEf0oFS4J7oc?input=http://www.eurogrand.com/
         {
-            "header_full": "http://www.finanse.mf.gov.pl/inne-podatki/podatek-od-gier-gry-hazardowe/komunikat",
+            "header_full": "https://www.finanse.mf.gov.pl/inne-podatki/podatek-od-gier-gry-hazardowe/komunikat",
```

Comment:
I think we want to keep both of the fingerprints so that we can also mark the older measurements as confirmed